### PR TITLE
Mk3 41

### PR DESCRIPTION
**Pull request by Klarence**
Adding padding mixin.

### CODE REVIEW

**Comment by taurean** [comment]
This should be merged into the mark-3 branch. Is this what this was based off of?

File: utilities/_spacing.scss [hunk 1]
```diff
@@ -0,0 +1,160 @@
+@mixin padding($padding-location: 'shorthand top right bottom left', $size...) {
+
+  @if $padding-location == number {
+      @error '`#{$padding-location}` is not a valid value for $padding-location. Please set $padding-location to a string value. ';
+  }
+  @if $padding-location == undefined {
+      @error '`#{$padding-location}` is not a valid value for $padding-location. Please set $padding-location to a string value. ';
+  }
```
Comment (taurean) [inline utilities/_spacing.scss]: This checks for `padding-location`s value type? that's cool

**Comment by taurean** [comment]
If this was branched off of mark-3, all that needs to change is it needs to be merged back into that branch. Looks good 👍. Did you test that it outputs the CSS correctly? I'm assuming it does.

**Comment by Klarence** [comment]
I tested it outputted correctly in sassmeister. But I should test it in a local  environment. 
Do you have any test pages?

**Comment by Klarence** [comment]
Yeah, it looks like I branched off of that branch. I'll update the MR.

**Comment by taurean** [comment]
I've just been using the testing environment repo personally.

**Pull request merged**